utils.css

```diff
@@ -10,6 +14,14 @@
     display: block;
 }
 
+.inline-block {
+    display: inline-block;
+}
+
+.align-text-center {
```

Comment:
Rename to `text-center`. The current name makes me think of the `align-items` property in flex.